setup.py

```diff
@@ -28,8 +26,6 @@
       packages=find_packages(),
       include_package_data=True,
       zip_safe=False,
-      install_requires=requires,
-      tests_require=requires,
```

Comment:
@rfk, what do you think about the removal of `install_requires` and `tests_require` parameters on the `setup()` call? It seems removing the `tests_require` results into more packages being installed than actually needed in the Travis job, as the `setup.py` gets used in the MAKEFILE.

I opted to remove these params, but now I'm wondering if it would have been better to create requirements.txt file without hash-checks and use it here. 🤔

Reply:
> what do you think about the removal of install_requires and tests_require parameters
> on the setup() call

I was surprised to see them go, but it doesn't worry me. As long as Travis installs and runs the tests somehow, seems fine.

> It seems removing the tests_require results into more packages being installed
> than actually needed in the Travis job

What's the underlying concern here, is it that this will make the Travis jobs run longer or be more expensive or something like that?

In general I think it would be good to have Travis install and use the same set of packages that will be installed for production use, just in case of any weird interactions that happen with one set of dependencies but not another (it's rare, but it can happen!).